packages/bpk-component-chip/examples.js

```diff
@@ -200,6 +268,10 @@ const DismissibleChipsExample = () => (
 
 const SelectableChipsExample = () => <StatefulSelectableChips />;
 
+const RadiogroupChipsExample = () => (
```

Comment:
```suggestion
const RadioGroupChipsExample = () => (
```

Reply:
Open question if it should be one or the other?

Reply:
React's own naming convention is with Ollie – `Radiogroup` isn't a valid word, so it should be `RadioGroup` to reflect that.